tests/kafkatest/services/streams.py

```diff
@@ -208,9 +208,27 @@ def __init__(self, test_context, kafka, eosEnabled):
                                                                 "org.apache.kafka.streams.tests.BrokerCompatibilityTest",
                                                                 eosEnabled)
 
+
 class StreamsBrokerDownResilienceService(StreamsTestBaseService):
     def __init__(self, test_context, kafka, configs):
         super(StreamsBrokerDownResilienceService, self).__init__(test_context,
                                                                  kafka,
                                                                  "org.apache.kafka.streams.tests.StreamsBrokerDownResilienceTest",
                                                                  configs)
+
+    def start_cmd(self, node):
```

Comment:
Where is this function used?

Reply:
when calling `processor.start()`.  It overrides the `streams.py#start_cmd` which in turn overrides the [Service.start_cmd](http://ducktape-docs.readthedocs.io/en/latest/_modules/ducktape/services/service.html#Service.start) 

This is needed due to one test starting with Kafka down.  The default implementation verifies Kafka is up, but we need to skip that for the `test_streams_runs_with_broker_down_initially`